### PR TITLE
[13.0][MIG] website_sale_require_login: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
Depends #336 

cc @tecnativa TT21705

### CODE REVIEW

File: website_sale_require_login/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+* Rafael Blasco <rafaelbn@antiun.com>
+* Jairo Llopis <yajo.sk8@gmail.com>
+* Dave Lasley <dave@laslabs.com>
+* Oscar Alcala <oscar@vauxoo.com>
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Alexandre Díaz <alexandre.diaz@tecnativa.com>
```
Comment (manuelcalerosolis) [inline website_sale_require_login/readme/CONTRIBUTORS.rst]: * `Tecnativa <https://www.tecnativa.com>`__:

  * Rafael Blasco 
  * Jairo Llopis
  * Alexandre Díaz

**Review by manuelcalerosolis — APPROVED** [review]

**Comment by Tardo** [comment]
Changes done

**Review by chienandalu — APPROVED** [review]
Squash last commit :)

**Pull request merged**